integral_view/forms/system_date_time_form.py

```diff
@@ -0,0 +1,15 @@
+from django import forms
+
+
+class date_time_form(forms.Form):
```

Comment:
All class names should be in camel case - so this should be called DateTimeForm().